cardano-config/src/Cardano/TracingOrphanInstances/Consensus.hs

```diff
@@ -321,10 +325,17 @@ instance (Condense (HeaderHash blk), LedgerSupportsProtocol blk)
           "Block fits onto the current chain: " <> condenseT pt
         ChainDB.TrySwitchToAFork pt _ -> \_o ->
           "Block fits onto some fork: " <> condenseT pt
-        ChainDB.AddedToCurrentChain _ _ c -> \_o ->
+        ChainDB.AddedToCurrentChain _ws _ _ c -> \_o ->
           "Chain extended, new tip: " <> condenseT (AF.headPoint c)
-        ChainDB.SwitchedToAFork _ _ c -> \_o ->
+--        Text.concat [ "\nWarning: " <> formatText w o | w <- ws ]
+--TODO: we could trace these warnings but then a
+-- HasTextFormatter (LedgerWarning blk) constraint will propagate everywhere
+-- and it's not clear that's how it's supposed to work, since we don't have any
+-- other HasTextFormatter constraints floating around
```

Comment:
I haven't looked at this in detail, but would `show` work for now?

Reply:
done